engine/events/pokecenter_pc.asm

```diff
@@ -39,12 +39,19 @@ PokemonCenterPC: ; 1559a
 	dw PlaceNthMenuStrings
 	dw .JumpTable
 
+PCPC_PLAYERS_PC   EQU 0
```

Comment:
``PCPC``? Is this a typo?

Reply:
"Pokémon Center PC".

Reply:
My bad, thought it was some form of find/replace gone wrong.